### ISSUE TITLE
Implement an `ObservationAggregator` observation function

### ISSUE THREAD

**Issue by gasse** [ds4dm/ecole#51]
In order to be able to combine several observation functions. The observations are then returned in a list, in the same order as the passed ObservationFunction's.

Pseudocode of the use case:
```
of = ObservationAggregator([NodeBipartite(), StrongBranchingScores()])
env = Environment(observation_function=of)

obs, done = env.reset()
bipartitegraph, sb_scores = obs
```

**Comment by AntoinePrv** [comment]
Part of #49

**Comment by AntoinePrv** [comment]
Implemented in d456802 currently in [`v0.2.x`](https://github.com/ds4dm/ecole/tree/v0.2.x).
